.gitignore

```diff
@@ -56,3 +56,6 @@ typings/
 
 # dotenv environment variables file
 .env
+
+# npm@5 unnecessary files
```

Comment:
this is not unnecessary. It might be that your current use-case installs this package with a version of npm < 5.0.0 but that doesn't mean the everyone is always going to use that version range.

pls undo